config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2074,6 +2074,10 @@
 # turnitin.report_gen_speed.setting.2=1
 # turnitin.report_gen_speed.setting.3=2
 
+# Allow Sakai TAs to be sent to Turnitin as Instructors so they can view TII reports
+# DEFAULT: false
+# turnitin.enrollAssistantsAsInstructors=true
```

Comment:
you may want to change this to snake case to be consistent with the other permissions

Reply:
agree